File: add/codesnippet/VisualBasic/m-system.data.datacolumn_7_1.vb
Private Sub AddColumn()
    Dim columns As DataColumnCollection = _
        DataSet1.Tables("Orders").Columns
 
    ' Add a new column and return it.
    Dim column As DataColumn = columns.Add( _
        "Total", System.Type.GetType("System.Decimal"))
    column.ReadOnly = True
    column.Unique = False
End Sub

File: add/codesnippet/CSharp/m-system.data.datacolumn_7_1.cs
    private void TestAndRemove(DataColumn colToRemove)
    {
        DataColumnCollection columns; 
        // Get the DataColumnCollection from a DataTable in a DataSet.
        columns = DataSet1.Tables["Orders"].Columns;
 
        if(columns.Contains(colToRemove.ColumnName))
        {
            columns.Remove(colToRemove);
        }
    }